cms/static/js/views/settings/main.js

```diff
@@ -366,6 +366,8 @@ define(['js/views/validation', 'codemirror', 'underscore', 'jquery', 'jquery.ui'
                                cachethis.setAndValidate(field, newVal);
                            }
                        });
+                       cmTextArea = this.codeMirrors[thisTarget.id].getInputField();
+                       cmTextArea.setAttribute('id', 'cm-textarea');
```

Comment:
Actually, I see now that this is for the Course Overview field, and there is only one such thing on the page. So ID uniqueness should not be a problem. But can you include course-overview in the ID? Perhaps course-overview-cm-textarea?